includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -79,17 +79,14 @@ public function __construct() {
 		// Listing renewal support.
 		WP_Job_Manager_Helper_Renewals::instance( $this );
 
+		// Recaptcha support.
+		WP_Job_Manager_Recaptcha::instance( $this );
```

Comment:
As suggested by PSalm, we should remove the parameter here

```suggestion
		WP_Job_Manager_Recaptcha::instance();
```

Reply:
Done in 14ce1f574fb9b218acc10b13c92237cb73fd5ca8